tests/Filesystem.php

```diff
@@ -25,8 +25,19 @@ public static function readFile(string $fileName): string
     /**
      * @return list<string>
      */
-    public static function find(string $pattern): array
+    public static function findRecursively(string $path, string $pattern): array
     {
-        return glob($pattern) ?: [];
```

Comment:
`glob()` doesn't work with streams, so I took the next best approach: iterators + fnmatch.